src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -456,6 +462,12 @@
 type: command
 short-summary: Creates a series of steps for building, testing and OS & Framework patching containers. Tasks support triggers from git commits and base image updates.
 examples:
+  - name: Create a contextless task.
+    text: >
+        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry --cmd bash -c /dev/null
```

Comment:
`-t hello-workd:{{.Run.ID}}` is to specify the image tag and is not relevant to this example.

Reply:
This looks like an invalid example.